### DISCUSSION TITLE
Lora training questions

### DISCUSSION THREAD

**Discussion by SPCell** [bmaltais/kohya_ss#1426] [Q&A]
Hello Kohya! Me and my friend have big troubles with training non-human characters. My friend is making a monster from Digimon (https://wikimon.net/Aero_V-dramon) while I'm making a humanoid demon from Doom Eternal (https://doom.fandom.com/wiki/Marauder). Both of us so far trained many various versions of Lora and Lycoris, but results are not satisfying at all. I got best results on AdamW8bit while my friend got them on Dadaptation. Is Stable Diffusion even suitable for training and generating non-human concepts? We tried various datasets and settings, maybe we missed something important? Maybe Prodigy optimizer will help? However, Prodigy optimizer is not available in Colab Lora trainer. Is there a way to integrate Prodigy into Colab?

**Comment by shadowlocked** [comment]
I have tried Prodigy today, but am unable to get it past the stage of producing generic black ladies from the early 20th century. Am doing the rounds in Reddit and Discord, begging a Kohya JSON. There are many optimizer arguments that seem essential to make Prodigy work at all, and apparently a dozen semi-documented no-nos for other settings, but there is no one place where a guide to Prodigy in Kohya can be found right now. Anyone willing to help, I would be most grateful.
